usaspending_api/awards/v2/data_layer/orm.py

```diff
@@ -225,7 +226,10 @@ def create_place_of_performance_object(db_row_dict):
             ("country_name", db_row_dict["_pop_country_name"]),
             ("county_name", db_row_dict["_pop_county_name"]),
             ("city_name", db_row_dict["_pop_city_name"]),
-            ("state_code", db_row_dict["_pop_state_code"]),
+            ("state_code", db_row_dict["_pop_state_code"] if db_row_dict.get("_pop_state_code", None)
```

Comment:
No need to use .get() since the keys will be present in the dict
```suggestion
            ("state_code", db_row_dict["_pop_state_code"] if db_row_dict["_pop_state_code"]
```

Reply:
In practice that has been shown not to be true. 45400393, the award in the ticket, is one such example, having only the state name.

Reply:
Using .get() is good to use when it is possible the key does not exist in the DB. If the key doesn't exist, instead of throwing a KeyError it will return None by default (or a provided value)

The original code guaranteed the key to be present in `db_row_dict`, so the value was obtained by `db_row_dict["_pop_state_code"]`. I didn't encounter any issues when I tested my recommended changes locally using `45400393`

Reply:
where does the code guarantee that?

Reply:
afaik `db_row_dict` is pulled from the `transaction` response in the `construct_assistance/contract_response`, which gets pulled from `FPDS_CONTRACT_FIELDS/FABS_ASSISTANCE_FIELDS`, both of which will always contain a `_pop_state_code` field, even if the field is NULL/None.